src/components/CreateTopic/CreateTopic.js

```diff
@@ -69,7 +69,7 @@ export default class CreateTopic extends Component {
   ignoreHash = name => name[0] === '#' ? name.slice(1) : name
 
   createAndNotify = name => {
-    this.props.createTopic(name, this.props.communityId)
+    this.props.createTopic(name, this.props.communityId, false, !!this.props.subscribeAfterCreate)
 
     // Note: assumes success
     this.setState({
```

Comment:
It's a small thing but whenever I touch a `setState` these days I convert it to a "functional setState" such that `setState({})` becomes `setState(prevState => ({}))` (or simply `setState(() => ({}))` in most use cases) . You are probably already aware of this and just updating an existing line of code and moving on, but thought it was worth mentioning. I don't know of a scenario in which not using a functional `setState` has bit me in the butt, but it would be a tough one to track down once it did. Better safe than sorry.